exercises/concept/meteorology/.docs/instructions.md

```diff
@@ -0,0 +1,35 @@
+# Instructions
+
+Your team is working on a meteorology application. 
+They have defined an API with various types and constants representing meteorological data, see file `meteorology.go`. 
+  
+Your task is to add suitable `String` methods to all types so that they implement interface `Stringer`. 
```

Comment:
Instead of asking for everything in one go, in Exercism we tend to prefer a more task-based approach for concept exercises, where each task should be simple. Tasks can of course build on each other and be increasingly more challenging (without being too much challenging, since this is a concept exercise).

The way you made the exercise is suitable for such a task division already. A suggestion for the tasks:

- Task 1: Implement interface for temperature unit
- Task 2: Implement the interface for temperature
- Task 3: Implement the interface for speed unit
- Task 4: Implement the interface for wind speed
- Task 5: Implement the interface for MetData

Each task is identified by having a `##` header with the number of the task. See [Blackjack](https://github.com/exercism/go/blob/main/exercises/concept/blackjack/.docs/instructions.md) for an example (there's 2 tasks in that exercise).

Separate tasks are also a great way to develop the story of the exercise a bit more, where you might go into more detail on why "the team" needs to format strings the way they do, or invent a funny situation related to temperatures.

Usually each task must also have its own test function, but you already have that, so no changes needed there.

The hints in `hints.md` must also be task-specific. In this case, it's ok to have some repetition between tasks in the hints, but the same `##` headers that are in the `instructions.md` must also be present in `hints.md`. This way the website knows how to associate the hints to each task and will display a button to show the hints next to each task.

Reply:
Thanks for this suggestion.  I experimented a bit with having three subtasks ([1,2], [3,4], [5]) but then I went with five tasks as suggested. Please feel free to add more "story-line" if you can think of something nice. 

Having the same "generic hint" appear five times under different headings `hints.md` is a big ugly but I guess it can't be helped?